### PR TITLE
Add kopano konnect and a caddy as proxy

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Setup the domain name in etch/hosts - assuming you are on linux

127.0.0.1 konnect.docker-playground.local
127.0.0.1 owncloud.docker-playground.local

```
KOPANO_KONNECT_DOMAIN=konnect.docker-playground.local
OWNCLOUD_DOMAIN=owncloud.docker-playground.local
docker-compose \
    -f owncloud-base.yml \
    -f owncloud-official.yml \
    -f cache/redis.yml \
    -f database/mariadb.yml \
    -f ldap/openldap.yml \
    -f ldap/openldap-mount-ldif.yml \
    -f ldap/openldap-autoconfig-base.yml \
    -f kopano/konnect/docker-compose.yml \
    up
```

https://konnect.docker-playground.local/.well-known/openid-configuration

web login: https://konnect.docker-playground.local/signin/v1/identifier
aaliyah_abernathy / secret

owncloud: https://owncloud.docker-playground.local

### CODE REVIEW

**Comment by michaelstingl** [comment]
> Setup the domain name in etch/hosts - assuming you are on linux

Local setup is not in my scope and no requirement for client dev and qa. Sorry.

**Comment by DeepDiver1975** [comment]
> Local setup is not in my scope and no requirement for client dev and qa. Sorry.

maybe not explicit enough - or setup dns in what ever way you want it to work.
etc/hosts is the simplest way to test this.

**Comment by michaelstingl** [comment]
> maybe not explicit enough - or setup dns in what ever way you want it to work.
> etc/hosts is the simplest way to test this.

Okay, I'll try it…

File: kopano/konnect/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Setup the domain name in etch/hosts - assuming you are on linux
```
Comment (xoxys) [inline kopano/konnect/README.md]: Typo, should be `/etc/hosts`

File: kopano/konnect/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+127.0.0.1 konnect.docker-playground.local
+127.0.0.1 owncloud.docker-playground.local
+
+```
```
Comment (xoxys) [inline kopano/konnect/README.md]: Please add language selector "```console"

File: kopano/konnect/konnectd-identifier-registration.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+
```
Comment (xoxys) [inline kopano/konnect/konnectd-identifier-registration.yaml:8]: Remove empty line and fix indentation (remove 4 spaces at the beginning of each line)

**Review by xoxys — CHANGES_REQUESTED** [review]
Just a few formatting :)

**Review by xoxys — APPROVED** [review]
LGTM

**Pull request merged**